app/views/shared/_message_controls.html.haml

```diff
@@ -12,7 +12,7 @@
     .grid-row.js-test-send-buttons
       Enter a message below and optionally attach a file (maximum 1 file per message).
       .message-column
-        = f.text_area :body, as: :text, rows: 5, placeholder: t('.message_placeholder'), label: false, class: 'form-control'
+        = f.text_area :body, as: :text, rows: 10, placeholder: t('.message_placeholder'), label: false, class: 'form-control'
```

Comment:
I think 10 rows might be too big. It hinders the ability of the user to see the feedback of the message sent/failed, as the text area will fill half of the window.
But probably we should also rethink how we show this feedback.